lib/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ForwardRefRenderFunction } from "react";
+import { textAreaStyle } from "./text-area.css";
+import { TextAreaProps } from "./text-area.types";
+
+const TextAreaComponent: ForwardRefRenderFunction<HTMLTextAreaElement, TextAreaProps> = ({ 
+    className,
+    children,
+    style,
+    ...nativeTextAreaProps
```

Comment:
for validation related sutffs, lets have:

`statusBorder` : (can be any color)

Reply:
also @Shadow-Flash is doing auto-resizing on textBox a hard task? eg: https://mui.com/material-ui/react-textarea-autosize/#main-content

if its not possible without any external library lets not look into it. Otherwise its something nice to have